apps/angular-thirty-seconds/src/app/angular-thirty-seconds/snippet-list/snippet-list.component.html

```diff
@@ -0,0 +1,10 @@
+<button routerLink="new">New</button>
+<div *ngFor="let snippet of snippets; let i = index">
```

Comment:
How do you think about move it to "...app/angular-thirty-seconds/snippet-demo/snippet-demo.component.html" to try to get rid of "i !== demoIndex"?

Reply:
Don't understand